packages/paste-core/components/alert-dialog/src/AlertDialog.tsx

```diff
@@ -11,8 +12,10 @@ import { AlertDialogContent } from "./AlertDialogContent";
 import { AlertDialogFooter } from "./AlertDialogFooter";
 import { AlertDialogHeader } from "./AlertDialogHeader";
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-const getAnimationStates = (): any => ({
+/**
+ * Animation states for the Alert Dialog.
+ */
+const AnimationStates = {
```

Comment:
It looks like we were attempting to avoid using the ReactSpring types here. I replaced the function with a hard coded object so that we weren't creating new objects each time this component ran. Further down in the component I converted from the SpringValue type to the native type so we don't have to use `any`.